.cirrus.yml

```
@@ -7,7 +7,7 @@ windows_docker_builder:
   os_version: 2019
   env:
     COMPUTE_ENGINE_IMAGE_NAME: windows-docker-builder-v20200813
-  pull_script: docker pull mcr.microsoft.com/windows/servercore:ltsc2019
-  build_core_script: docker build --tag cirrusci/windowsservercore:2019 windowsservercore
-  build_cmake_script: docker build --tag cirrusci/windowsservercore:cmake contrib/cmake
+  pull_script: docker pull mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2019
+  build_core_script: docker build -m 2GB --tag cirrusci/windowsservercore:2019 windowsservercore
+  build_cmake_script: docker build -m 2GB --tag cirrusci/windowsservercore:cmake contrib/cmake
```

Is it possible to pass storage options here?

```suggestion
  build_cmake_script: docker build --storage-opt "size=50GB" -m 2GB --tag cirrusci/windowsservercore:cmake contrib/cmake
```

---

To my knowledge `docker build` does not support such an option. Locally I edited the configuration for the docker daemon, but I have no idea how/if the docker build host for this repository can be changed/supports it.